mubench.pipeline/tasks/implementations/checkout.py

```diff
@@ -31,7 +34,15 @@ def process_project_version(self, project: Project, version: ProjectVersion) ->
                 logger.debug("Already checked out %s.", version)
             else:
                 logger.info("Fetching %s from %s...", version, checkout)
-                checkout.create()
+                if self.use_temp_dir:
+                    temp_dir = mkdtemp(prefix="mubench-checkout_")
+                    temp_checkout = version.get_checkout(temp_dir)
+                    temp_checkout.create()
```

Comment:
For the repository checkout this means that we clone from the remote for every version, since the base checkout doesn't exist in the temp directory, right? Not sure we can do anything about that... so I guess I could live with it, seeing that this is only a Windows workaround and checkout doesn't happen frequently...